tools/plcquery.py

```diff
@@ -145,6 +145,7 @@ def parse_options():
 
     parser.set_defaults(
                         action="get",
+                        session_timeout=2592000, # 30 days (60*60*24*30)
```

Comment:
You're right! I added a commit which creates a new global variable `SESSION_TIMEOUT=60*60*24*30`, which is less hard coding deep in the file and also more easily understood at a glance.